[2025-01-31T04:46:46.679+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2022-08-18T00:00:00+00:00 [queued]>
[2025-01-31T04:46:46.695+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v2.second_task scheduled__2022-08-18T00:00:00+00:00 [queued]>
[2025-01-31T04:46:46.696+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T04:46:46.724+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2022-08-18 00:00:00+00:00
[2025-01-31T04:46:46.734+0000] {standard_task_runner.py:57} INFO - Started process 4284 to run task
[2025-01-31T04:46:46.741+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v2', 'second_task', 'scheduled__2022-08-18T00:00:00+00:00', '--job-id', '1172', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpolslfr95']
[2025-01-31T04:46:46.743+0000] {standard_task_runner.py:85} INFO - Job 1172: Subtask second_task
[2025-01-31T04:46:46.917+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v2.second_task scheduled__2022-08-18T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T04:46:47.160+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-08-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-18T00:00:00+00:00'
[2025-01-31T04:46:47.163+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T04:46:47.164+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the second task, I run after the first task']
[2025-01-31T04:46:47.186+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T04:46:47.189+0000] {subprocess.py:93} INFO - I am the second task, I run after the first task
[2025-01-31T04:46:47.191+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T04:46:47.238+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v2, task_id=second_task, execution_date=20220818T000000, start_date=20250131T044646, end_date=20250131T044647
[2025-01-31T04:46:47.292+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T04:46:47.323+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
